CONTRIBUTING.md

```diff
@@ -245,3 +245,34 @@ npm run update_prebuilt
 
 That last step is easy to forget, so try to make it a habit whenever you're
 prepping a PR that includes frontend work.
+
+### MacOS
+
+Install Xcode and Homebrew Mac. Once that is done: 
+
+```bash
+#Install dependent packages
```

Comment:
nit: To me, a space between the # and the comment text makes comments more readable.
nit+: I also like comments to be full sentences(ish) with capitalization and punctuation.

Google does have a [shell style guide](https://google.github.io/styleguide/shell.xml#Comments), but it doesn't explicitly verify my nits that I can see (though its examples are consistent with my nits).